src/processor/Package.ts

```diff
@@ -330,4 +331,63 @@ export class Package {
       pullAt(sd.rules, rulesToRemove);
     });
   }
+
+  private removeDateRules(): void {
+    let allDatesMatch = true;
+    let date: string;
+    [...this.profiles, ...this.extensions, ...this.valueSets, ...this.codeSystems].forEach(sd => {
```

Comment:
Maybe instead of `sd` we could call this `resource`? Since `valueSets` and `codeSystems` are not `sd`.